### DISCUSSION TITLE
Are there any plans to include a function to STDOUT STDIN SQL in a fixed format?

### DISCUSSION THREAD

**Discussion by friendbear** [TaKO8Ki/gobang#124] [Q&A] [answered]
💡  My idea, but it's only a TUI tool.

I thought it would be nice to be able to hit it with a different command.

* That way, we can release two versions.
However, I think it will be a pain if the core part is not designed to be separated.

**Comment by TaKO8Ki** [comment, marked as answer]
I don't understand much of this, so could you please provide a specific example?

**Reply by friendbear** [reply]
In the case of Oracle, there is a command called sqlplus.
It is very versatile, so you don't need to copy it, but you can imagine it as

```sh
sqlplus -s admin/password << EOF
whenever sqlerror exit sql.sqlcode;
set echo off 
set heading off

@pl_script_1.sql
@pl_script_2.sql

exit;
EOF
````



The following is an example of how to use sqlplus in Python to pass SQL to standard input and output from standard output.

* <https://www.bobbydurrettdba.com/2016/11/04/running-sqlplus-from-a-python-script/>
